hw/xbox/nv2a/nv2a.c

```diff
@@ -328,11 +327,11 @@ static void nv2a_realize(PCIDevice *dev, Error **errp)
     vga->msr = VGA_MIS_COLOR;
 
     vga_common_init(vga, OBJECT(dev), false); // FIXME: true or false? idk
-    vga->get_bpp = nv2a_get_bpp;
+    vga->get_bpp     = nv2a_get_bpp;
```

Comment:
This and next block also don't need alignment. It just groups unrelated things which does not benefit readibility.